### PR TITLE
Use qualified module name from diagnostics in suggestNewImport

### PR DESCRIPTION
**Pull request by berberman**
GHC helps us to split the qualified module name from the identifier. Closes haskell/haskell-language-server#629

### CODE REVIEW

**Comment by pepeiborra** [comment]
Haven't looked at the code yet, but haskell/haskell-language-server#629 is about operators and the title of this PR "Use qualified module name in suggestNewImport" doesn't seem related.

**Comment by berberman** [comment]
Ah, the issue was caused by we mistakenly stripped the qualified import prefix of identifiers. For example, if I write `T.null` , where the `T` is not in scope, diagnostic will report:

> Not in scope: ‘T.null’
> No module named ‘T’ is imported.

The previous logic only took the first line, then tried to use `splitOn "."` to strip `T.`, obtaining the identifier `null`, which we use to lookup the export map. Everything goes well.

But operators like `.|.` were split unexpectedly, result in `|.` as the identifer. In this PR, I try to extract the qualified module name from the second line of the diagnostic message, so we no longer to use `splitOn` and `intercalate` manually, removing the qualified module name by `T.stripPrefix` instead.

**Comment by berberman** [comment]
Looks like the CI broke for some unknown reasons, but all checks passed on my fork: https://github.com/berberman/ghcide/tree/patch-qual
Ready to be reviewed and merged :-)

**Review by pepeiborra — APPROVED** [review]
Thank you, this is an awesome bug fix.  

I don't know what the problem is with CI, could you perhaps rebase on top of master?

File: src/Development/IDE/Plugin/CodeAction.hs [hunk 1]
```diff
@@ -624,7 +624,10 @@ suggestExtendImport exportsMap contents Diagnostic{_range=_range,..}
             Just result <- addBindingToImportList ident importLine
             = [("Add " <> renderIdentInfo ident <> " to the import list of " <> mod, [TextEdit range result])]
           | otherwise = []
-        lookupExportMap binding mod 
+        renderImport IdentInfo {parent, rendered}
```
Comment (berberman) [inline src/Development/IDE/Plugin/CodeAction.hs:627]: haha, it came back

Comment (pepeiborra) [reply src/Development/IDE/Plugin/CodeAction.hs]: That was my fault, trying to fix a merge conflict

**Pull request merged**